README.md

```diff
@@ -44,7 +44,25 @@ $ slack create my-app -t slack-samples/deno-starter-template
 $ cd my-app
 ```
 
-## Create a Link Trigger
+## Running Your Project Locally
+
+While building your app, you can see your changes propagated to your workspace
+in real-time with `slack run`. You'll know an app is the development version if
+the name has the string `(local)` appended.
+
+```zsh
+# Run app locally
+$ slack run
+
+Connected, awaiting events
+```
+
+To stop running locally, press `<CTRL> + C` to end the process.
+
+## Creating Triggers
+
+When you run or deploy your project, the CLI will prompt you to create a new
+trigger if one is found in the `triggers/` directory.
```

Comment:
I think this behaviour also only happens if no trigger exists for the particular app instance (app+workspace).

Worth calling that out here or no?

Reply:
Oh, really? So, if someone evolves the app and later adds a trigger, it won't get picked up?

Reply:
Did a bit of editing and rearranging. Let me know what you think, but the nod added is:

> When you `run` or `deploy` your project for the first time, the CLI will prompt
you to create a trigger if one is found in the `triggers/` directory. For any
subsequent triggers added to the application, each must be
[manually added using the `trigger create` command](#manual-trigger-creation).

Reply:
Perfect, thank you!